### PR TITLE
Fixing diagrams for sign off gr

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc [hunk 1]
```diff
@@ -97,7 +97,7 @@ In Local Mode, Flex Gateway typically acts as an ingress controller that manages
 
 When using this mode, you must know what resources to create and apply, and use `kubectl apply` to deploy Kubernetes targets and resources such as APIs, policies, or Flex Gateway-related resources to your gateway. 
 
-Local Mode also supports the Ingress resources from Kubernetes, which are configured through a YAML file with properties, such as `apiVersion`, `kind`, `metadata`, `spec` for configuring a load balancer or proxy server, and `rules` for directing HTTP and HTTPS traffic (see https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes documentation^] on Ingress resources).
+Local Mode also supports `Ingress` resources from Kubernetes, which are configured through a YAML file with properties, such as `apiVersion`, `kind`, `metadata`, `spec` for configuring a load balancer or proxy server, and `rules` for directing HTTP and HTTPS traffic. For more information about Flex Gateway managing `Ingress` resources, see xref:flex-gateway-k8-ingress-class[].
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc]:
```suggestion
Local Mode also supports `Ingress` resources from Kubernetes, which are configured through a YAML file with properties, such as `apiVersion`, `kind`, `metadata`, and `spec` for configuring a load balancer or proxy server, and `rules` for directing HTTP and HTTPS traffic. For more information about how Flex Gateway manages `Ingress` resources, see xref:flex-gateway-k8-ingress-class[].
```

File: gateway/1.4/modules/ROOT/pages/flex-architecture-basic-deployments.adoc [hunk 2]
```diff
@@ -13,18 +13,22 @@ You can extend the deployment models in this overview to all technological stack
...
 
 image:ad-standalone-deployment.png["In a standalone deployment, Flex Gateway and the consumer applications are in different Kubernetes namespaces."]
 
-As the diagram shows, all external traffic in a standalone deployment passes through Flex Gateway before reaching the consumer APIs.
+
 
 == Sidecar Deployment
 In a sidecar deployment, each Flex Gateway deployment exclusively protects the APIs exposed by its protected service. A new Flex Gateway replica is added with each new protected service:
 
+As the following diagram shows, external traffic in a sidecar deployment passes through Flex Gateway and then to their respective consumer API. Additionally, the consumer application can belong to the same network as Flex Gateway.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-architecture-basic-deployments.adoc]:
```suggestion
As the following diagram shows, external traffic in a sidecar deployment passes through Flex Gateway to the respective consumer API. The consumer application can belong to the same network as Flex Gateway.
```

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-overview.adoc [hunk 3]
```diff
@@ -24,8 +24,10 @@ include::partial$task-reg-run-flex-gateway.adoc[tag=k8s-connected-intro]
 == Local Mode with Kubernetes
 include::partial$task-reg-run-flex-gateway.adoc[tag=k8s-local-intro]
 
-== See Also
+The following diagram demonstrates multiple Flex Replicas deployed as ingress controllers. The Flex Replicas sits behind a load balancer and manage all incoming traffic.
```
Comment (hannanelson) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-overview.adoc]:
```suggestion
The following diagram demonstrates multiple Flex Replicas deployed as ingress controllers. The Flex Replicas sit behind a load balancer and manage all incoming traffic. 
```

Comment (hannanelson) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-overview.adoc]: Should "Replica" be lowercase? Or is it a proper noun?

**Review by hannanelson — APPROVED** [review]

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-overview.adoc (hunk 3 above)
Comment (glenn-rodgers-sf) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-overview.adoc]: The capitalization is correct. It's a proper noun, or at least we refer to it as one.

**Review by hannanelson — APPROVED** [review]

**Pull request merged**